paddle/fluid/operators/fc_op.cc

```diff
@@ -128,6 +119,12 @@ void FCOpMaker::Make() {
   AddAttr<bool>("use_mkldnn",
                 "(bool, default false) Only used in mkldnn kernel")
       .SetDefault(false);
+  AddAttr<bool>(
+      framework::kAllKernelsMustComputeRuntimeShape,
+      "If an Op has this attribute, all its kernels should calculate output"
```

Comment:
Maybe it's not necessary adding these comments,  all op adding these should be duplicated, or comments should be simple.

Reply:
Done, simplify the comments.